src/index.ts

```diff
@@ -180,36 +182,36 @@ export function buildCallsContext(associatedResults: AssociatedCallResult[]) {
  * @returns T
  */
 export function decodeMetaResults(
-  metaResults: CallResult[],
-  context: CallContext[]
+    metaResults: CallResult[],
+    context: CallContext[]
 ): MetaByContract {
-  return metaResults.reduce((meta: BalancesByContract, result, index) => {
-    let methodValue;
-    const { 1: data } = result;
-    const { contractAddress, methodName } = context[index];
-
-    try {
-      const type = fragmentTypes[methodName];
-      [methodValue] = coder.decode([type], data);
-    } catch (error) {
-      console.info(
-        `Problem decoding ${methodName} for ${contractAddress}. The contract is likely not ERC-20 compliant.`
-      );
-      methodValue = decodeBytes32String(data);
-    }
-
-    if (methodName === "decimals") {
-      methodValue = Number(methodValue);
-    }
-
-    return {
-      ...meta,
-      [contractAddress]: {
-        ...meta[contractAddress],
-        [methodName]: methodValue,
-      },
-    };
-  }, {});
+    return metaResults.reduce((meta: BalancesByContract, result, index) => {
+        let methodValue;
+        const {1: data} = result;
+        const {contractAddress, methodName} = context[index];
+
+        try {
+            const type = fragmentTypes[methodName];
+            [methodValue] = coder.decode([type], data);
+        } catch (error) {
+            console.info(
+                `Problem decoding ${methodName} for ${contractAddress}. The contract is likely not ERC-20 compliant.`
+            );
+            methodValue = decodeBytes32String(data);
+        }
+
+        if (methodName === "decimals") {
+            methodValue = Number(methodValue);
+        }
+
+        return {
+            ...meta,
+            [contractAddress]: {
+                ...meta[contractAddress],
+                [methodName]: methodValue,
+            },
+        };
+    }, {});
```

Comment:
It appears that some additional formatting has been inadvertently added. Could these be removed? I think adding a Prettier config should be handled as a separate future task. Thanks!